lib/main_common.pm

```diff
@@ -1644,7 +1644,7 @@ sub load_filesystem_tests {
         # kernel module required by thin-LVM
         loadtest 'console/snapper_thin_lvm' unless is_jeos;
     }
-    loadtest 'console/snapper_used_space';
+    loadtest 'console/snapper_used_space' if is_sle('15-SP1+');
```

Comment:
what about openSUSE, especially TW? Please keep in mind to start testing features always with openSUSE TW.

Reply:
you're right! It was already working for TW and Leap15.1
#6017

Reply:
you're right, it was even working for tw and leap15.1.
#6017

Reply:
you're right, it was even working for tw and leap15.1.
#6017